toqito/channel_ops/choi_to_kraus.py

```diff
@@ -12,7 +12,7 @@ def choi_to_kraus(
     choi_mat: np.ndarray, tol: float = 1e-9, dim: int | list[int] | np.ndarray = None
 ) -> list[np.ndarray] | list[list[np.ndarray]]:
     r"""
-    Compute a list of Kraus operators from the Choi matrix [Rigetti20]_.
+    Compute a list of Kraus operators from the Choi matrix from `Rigetti <https://github.com/rigetti/forest-benchmarking>`__.
```

Comment:
We have a link to the forest benchmarking docs here but elsewhere we have a reference to the bibtex key for forest. Either is fine, but should be consistent.